### DISCUSSION TITLE
How can i get the Datatime of a error?

### DISCUSSION THREAD

**Discussion by contibru** [cypress-io/cypress#17482] [General]
I looked in the docs and search the web but I don`t found a way to show the exactly date time of an error on cypres.

For example:
![image](https://user-images.githubusercontent.com/30509284/126986061-72677d92-4bd6-4c3e-9225-67a36ea287f1.png)

Ok, there is a error, but what time it occurred?
With this information i am capable of look into logs of the application.

**Comment by bahmutov** [comment]
See the solution in https://glebbahmutov.com/blog/cypress-tips-and-tricks/#print-timestamp-with-the-error-message

**Reply by contibru** [reply]
Thank you, it is exactly what I`m looking for.
